tests/requirements.txt

```diff
@@ -1,4 +1,7 @@
-pytest>=6.1.1,<7.0.0
-ledgercomm>=1.1.0,<1.2.0
+pytest
+ragger[speculos]
+ragger[ledgerwallet]
+Jinja2==3.1.2
+Flask==2.1.2
```

Comment:
Why?

Reply:
CI failed otherwise while trying to install deps... Forcing the correct versions fixes it
Not sure if there is a cleaner fix